.pylintrc

```diff
@@ -0,0 +1,12 @@
+[MESSAGES CONTROL]
+disable=bad-continuation,         ######## Disabled because of conflicts with black ########
+        consider-using-enumerate, ######## Temporary disabled ########
+        import-error,
+        no-name-in-module,
+        too-many-statements,
+        unused-variable,
+
```

Comment:
I thought this file was already in master?

Reply:
Me too. I will do it afterward. Let's merge those first.